source/_integrations/vacuum.xiaomi_miio.markdown

```diff
@@ -179,6 +179,35 @@ automation:
           - 26496
 ```
 
+### Service `vacuum.send_command`
```

Comment:
This service works with this integration, but is not of this integration (it belongs to the `vacuum` integration) and should not be documented or listed on this page.

I would suggest adding an additional section with examples in this case.